diff -urN cpufreqd-1.0.1.org/Makefile.am cpufreqd-1.0.1/Makefile.am
--- cpufreqd-1.0.1.org/Makefile.am	2003-10-13 16:25:46.000000000 +0200
+++ cpufreqd-1.0.1/Makefile.am	2003-10-13 16:26:17.000000000 +0200
@@ -42,7 +42,7 @@
 	if [ -f $(DESTDIR)/$(sysconfdir)/cpufreqd.conf ]; then \
 	 echo "** preserving old configuration file **"; \
 	else \
- 	 install -Dm 0644 -o root -g root cpufreqd.conf $(DESTDIR)/$(sysconfdir)/cpufreqd.conf; \
+ 	 install -Dm 0644 cpufreqd.conf $(DESTDIR)/$(sysconfdir)/cpufreqd.conf; \
 	fi;
 
 #sysconf_DATA = cpufreqd.conf
