diff -urN cpufreqd-1.0-beta1.org/Makefile.am cpufreqd-1.0-beta1/Makefile.am
--- cpufreqd-1.0-beta1.org/Makefile.am	2003-06-15 12:08:01.000000000 +0000
+++ cpufreqd-1.0-beta1/Makefile.am	2003-06-15 12:08:15.000000000 +0000
@@ -30,11 +30,4 @@
 man1_MANS = manpages/cpufreqd.1
 man5_MANS = manpages/cpufreqd.conf.5
 
-install-data-local:
-	if [ -f /etc/cpufreqd.conf ]; then \
-	 echo "** preserving old configuration file **"; \
-	else \
- 	 install -m 0644 -o root -g root cpufreqd.conf /etc; \
-	fi;
-
 #sysconf_DATA = cpufreqd.conf
